user/trusty-ci-environment.md

```diff
@@ -4,233 +4,225 @@ layout: en
 permalink: /user/trusty-ci-environment/
 ---
 
-## tl;dr - Using Trusty
+## Using Trusty
 
-If you just want to get started using our Trusty beta, then add the following to your `.travis.yml`.
+In order to use Ubuntu Trusty, add the following to your `.travis.yml`.
 
 ``` yaml
+dist: trusty
 sudo: required
+```
+
+Or, if you want to route to the container-based beta, add:
+
+``` yaml
 dist: trusty
+sudo: false
+group: beta
 ```
 
 This is enabled for both public and private repositories.
 
-If you'd like to know more about the pros, cons, and current state of
-the Trusty beta, read on ...
+If you'd like to know more about the pros, cons, and current state of using
+Trusty, including details specific to the container-based beta, read on ...
 
 ### What This Guide Covers
 
-This guide explains what packages, tools and settings are available in
-the Trusty beta environment.
-
-A couple of important points to note:
-
-- All Trusty images have `sudo` enabled, so you're able to make any
-  changes you wish to the images using our various `before_*:`
-  keys or custom scripts.
-
-- The Trusty images are what our [Docker beta](/user/docker/)
-  runs on as well.
+This guide provides a general overview of which packages, tools and settings are
+available in the Trusty beta environment.
 
 <div id="toc"></div>
 
 ## Overview
 
-Travis CI runs each Trusty build in an isolated [Google Compute Engine](https://cloud.google.com/compute/docs/) virtual machine that offer a vanilla build
-environment for every build.
+### Fully-virtualized via `sudo: required`
+
+When specifying `sudo: required`, Travis CI runs each build in an isolated
+[Google Compute Engine](https://cloud.google.com/compute/docs/) virtual machine
+that offer a vanilla build environment for every build.
 
 This has the advantage that no state persists between builds, offering a clean
 slate and making sure that your tests run in an environment built from scratch.
 
 Builds have access to a variety of services for data storage and messaging, and
 can install anything that's required for them to run.
 
+### Container-based with `sudo: false`
 
-## Image differences from Precise
+When specifying `sudo: false`, Travis CI runs each build in a container on a
+shared host via Docker.  The container contents are a pristine copy of the
+Docker image, as guaranteed by Docker itself.
+
+The advantage of running with `sudo: false` is dramatically reduced time between
+commit push to GitHub and the start of a job on Travis CI, which works
+especially well when one's average job duration is under 3 minutes.
 
-In our Precise based environments, we've traditionally built a library
-of images based on common language runtimes like `ruby`, `go`, `php`,
-`python`, etc.
+The primary disadvantage of running with `sudo: false` is the unavailability of
+passwordless `sudo` rights.  In order to account for this disadvantage, we offer
+addons [such as `apt.packages` and
+`apt.sources`](/user/installing-dependencies/).
```

Comment:
can we add a warning here about the limited number of packages available on the Trusty on container beta? and what will come in the future to fix this (if applicable)?